modules/retail/pages/retail-overview.adoc

```diff
@@ -2,6 +2,10 @@ ifndef::backend-pdf[]
 [[retail-overview]]
 = Retail Guide
 
+ifeval::[{mlm-content} == true]
```

Comment:
uyuni-content == true

Reply:
Remember if this is uyuni, we do not want to index mlm pages. So, noindex on uyuni-content.